analyzers/golang/resolver/lockfile.go

```diff
@@ -30,7 +30,7 @@ const (
 func FromLockfile(tool Type, dir string) (Resolver, error) {
 	switch tool {
 	case Dep:
-		return dep.New(dir)
+		return dep.New(dir+"/Gopkg.lock", dir+"/Gopkg.toml")
```

Comment:
Use `filepath.Join`.

Reply:
Thank you for the tip, I haven't used that before.